src/components/Movie.js

```diff
@@ -30,10 +34,11 @@ function Movie() {
         return () => window.removeEventListener("resize", updateWindowDimensions) 

```

Comment:
this seems to me some sort of hack to take care of some bug

Reply:
no sir, its just to maintain the number of cards to render according to the screen size